tools/lint.js

```diff
@@ -66,7 +66,8 @@ async function dlintPreferPrimordials() {
   const sourceFiles = await getSources(ROOT_PATH, [
     "runtime/**/*.js",
     "ext/**/*.js",
-    "core/**/*.js",
+    "core/*.js",
```

Comment:
I don't know why `core/*.js` is OK but `core/**/*.js` is NG.

Reply:
Apparently globstar (`**`) has not agreed on the behavior and `git` requires one directory.